config/settings.py

```
@@ -151,17 +178,31 @@
-    BYTE_PATROLALLOWED_IPS = [
-    '127.0.0.1',
-    '69.255.206.86'
-    ]
-    BYTE_PATROLPROTECTED_PATHS = [
-    r'^/admin/.*$',    
-    r'^/checkup/?.*$'
-]
-    BYTE_PATROLCACHE_TIMEOUT = 9000 
 
+    @staticmethod
+    def _get_env_list(var_name, default):
```

Ruff (0.11.9)

Missing return type annotation for staticmethod `_get_env_list`

(ANN205)